docs/docs/guides/web3_config/index.md

```diff
@@ -30,6 +30,55 @@ There is list of configuration params that can be set for modifying behavior of
 
 ## Global level Config
 
+`handleRevert` is a configuration option that specifies whether the web3 provider should handle revert errors automatically. When set to true, the web3 provider will catch revert errors and return a more informative error message instead of throwing an exception.
```

Comment:
@MeenuyD Thanks for working on this, I'll suggest to create a new section in this guide, for explaning config options, and we need desc of all of following config params:
<img width="391" alt="image" src="https://github.com/web3/web3.js/assets/86780488/84d19b0a-26d6-415e-8ea2-b36dd75a3ce1">

Reply:
Ok

Reply:
@MeenuyD let us know when updates are complete, thx